docs/sphinx/user-guide/release-notes/2.5.x.rst

```diff
@@ -101,6 +101,15 @@ You can see the complete list of bugs that were
 Upgrade Instructions for 2.4.x --> 2.5.x
 ----------------------------------------
 
+.. warning::
+  If you have SELinux in Enforcing mode and you forget to stop pulp_workers, pulp_celerybeat, or
```

Comment:
pulp_workers, pulp_celerybeat, and pulp_resource_manager should be escaped I think.